3p/ampcontext.js

```diff
@@ -109,7 +109,7 @@ export class AmpContext {
    */
   setupMetadata_() {
     try {
-      const data = JSON.parse(decodeURI(this.win_.name));
+      const data = JSON.parse(decodeURI(this.client_.getWindow().name));
```

Comment:
sorry, do you mean a public property on iframemessagingclient, another private property on ampcontext, or something else?